app/frontend/src/features/communities/events/EventPage.tsx

```diff
@@ -63,6 +69,12 @@ export const useEventPageStyles = makeStyles((theme) => ({
   eventTitle: {
     gridArea: "eventTitle",
   },
+  onlineInfoContainer: {
+    display: "grid",
+    columnGap: theme.spacing(2),
+    gridAutoFlow: "column",
+    gridTemplateColumns: "max-content max-content",
```

Comment:
Is this the same as 1fr 1fr?

Reply:
Not quite, `1fr 1fr` would stretch that across the width of the title container and so leaves a big gap between "virtual event" and the "event link" text. I could have probably use flexbox for this... but I think I'm slowly becoming a fan of using `columnGap`/`rowGap` for defining spacing between child elements in a container.